tests/python/relay/test_backend_graph_executor.py

```diff
@@ -361,21 +361,21 @@ def test_benchmark_end_to_end(dev, target):
     assert len(result.results) == 2
 
 
-@tvm.testing.requires_llvm
+@tvm.testing.requires_cuda
 def test_benchmark_end_to_end_rpc():
     server = rpc.Server("127.0.0.1")
     remote = rpc.connect(server.host, server.port)
 
     mod, params = mlp.get_workload(1)
-    lib = relay.build(mod, target="llvm", params=params)
+    lib = relay.build(mod, target="cuda", params=params)
```

Comment:
why this change?

Reply:
The check that prevents you from copying from local to remote only allows copying from local cpu to remote cpu, so this test was not having any issues even though its behavior was wrong. (https://github.com/apache/tvm/pull/9175/files#diff-c2d3e12882a0e96941e57203f5f4c40eb7d9cf2e913d991fe8307f0930c2bb82R234-R238).

Reply:
ah gotcha.